### PR TITLE
Reduce line-height on titles [YEP-61]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@smn @bruskiza @miltontony tiny review please

**Comment by smarzban** [comment]
Ping @bruskiza @miltontony

File: tuneme/static/css/smart.css [hunk 1]
```diff
@@ -114,8 +114,8 @@ body {background:white; position:relative; text-align:center;}
 body {font:normal 17px 'Quicksand', Arial, Helvetica, sans-serif; color:#828282;}
 h1 { color:white; font-size:200%; }
     h2 a {color:white; text-decoration:none;}
-h3 {padding:10px 15px;}
-h4 {padding:10px 15px;}
+h3 {padding:10px 15px; line-height:1;}
+h4 {padding:10px 15px; line-height:1;}
```
Comment (miltontony) [inline tuneme/static/css/smart.css]: Please specify a unit - either `em` or `px`

**Comment by smarzban** [comment]
@miltontony done

**Comment by miltontony** [comment]
:+1:

**Pull request merged**